examples/log_parsing/postprocessing.py

```diff
@@ -74,7 +73,10 @@ def supports_cpp_node(self):
         return False
 
     def accepted_types(self) -> typing.Tuple:
-        return (MultiResponseLogParsingMessage, )
+        return (MultiPostprocLogParsingMessage, )
```

Comment:
This was just a bug in the original code. 
As far as I can tell what is going on is:
1) `LogParsingInferenceStage` incorrectly reports it's output type as `MultiResponseLogParsingMessage`
2) Similarly `LogParsingPostProcessingStage` incorrectly reports it's input type as `MultiResponseLogParsingMessage` also. 

At build time Morpheus checks that these both match, which they do and the pre-flight check passes. At runtime this isn't an issue since everything is Python. 

I believe I only caught this just as a function of noticing the error. I'm not sure if this can be really improved upon for Python-only stages.

Reply:
Yeah we'd necessarily take a performance hit if we tried to enforce the actual values coming across. Interesting.